orderer/consensus/smartbft/util.go

```diff
@@ -283,6 +283,20 @@ func (ri *RequestInspector) RequestID(rawReq []byte) types.RequestInfo {
 	if err != nil {
 		return types.RequestInfo{}
 	}
+
+	if req.chHdr.Type == int32(cb.HeaderType_CONFIG) {
+		configEnvelope := &cb.ConfigEnvelope{}
+		_, err = protoutil.UnmarshalEnvelopeOfType(req.envelope, cb.HeaderType_CONFIG, configEnvelope)
```

Comment:
why do you do this here? we already know the envelope is well formed, otherwise how did we do the check in line 287?